synapse/lib/storm.py

```diff
@@ -877,6 +913,37 @@ def _stormOperRefs(self, query, oper):
                     if limt.dec(len(news)):
                         break
 
+    def _stormOperAddNode(self, query, oper):
+        # addnode(<form>,<valu>,**props)
+        args = oper[1].get('args')
+        if len(args) != 2:
+            raise SyntaxError('addnode(<form>,<valu>,[<prop>=<pval>, ...])')
+
+        kwlist = oper[1].get('kwlist')
+
+        core = self.getStormCore()
+
+        props = dict(kwlist)
+
+        node = self.formTufoByProp(args[0],args[1],**props)
+        query.add(node)
+
+    def _stormOperDelNode(self, query, oper):
+        # delnode()
+        opts = dict( oper[1].get('kwlist') )
+
+        core = self.getStormCore()
+
+        force,_ = core.getTypeNorm('bool', opts.get('force',0))
+
+        nodes = query.take()
+        if force:
+            [ core.delTufo(n) for n in nodes ]
```

Comment:
So just calling delnode does nothing (force=True is required for it to delete the node).